### ISSUE TITLE
Cannot create project using leanproject new name, with elan 0.10.3

### ISSUE THREAD

**Issue by Kariiem** [Kha/elan#24]
### elan version
`PS C:\Users\Karim> elan --version`
`elan 0.10.3 (4687a92d0 2021-01-15)`

I updated using `elan self update` command.

### description
`leanproject new preoject-name`  throws exception.

`PS C:\Lean Workspace> leanproject new myproject
uncaught exception: Lean package manager, version 4.0.0

Usage: leanpkg <command>

configure              download and build dependencies and print resulting LEAN_PATH
build [-- <Lean-args>] configure and build *.olean files
init <name>            create a Lean package in the current directory

See `leanpkg help <command>` for more information on a specific command.
Command '['leanpkg', 'new', 'project']' returned non-zero exit status 1.
`
I tried with `leanpkg new projectname`, it also faild with the same error message.

`PS C:\Lean Workspace> leanpkg new myproject`
`uncaught exception: Lean package manager, version 4.0.0`

`Usage: leanpkg <command>`

`configure              download and build dependencies and print resulting LEAN_PATH`
`build [-- <Lean-args>] configure and build *.olean files`
`init <name>            create a Lean package in the current directory`

`See `leanpkg help <command>` for more information on a specific command.`

###Additional info
Installed toolchain:

PS C:\Users\Karim> elan toolchain list
stable
leanprover-community-lean-3.24.0
leanprover-lean4-stable (default)

**Comment by Kha** [comment]
As the error message says, there is no `new`. Use `init`.

**Comment by Kariiem** [comment]
This the output on using `leanproject init test`.
PS C:\Users\Karim\Desktop\Workspaces\Lean Workspace> leanproject init test
Usage: leanproject [OPTIONS] COMMAND [ARGS]...
Try 'leanproject -h' for help.

`Error: No such command 'init'.`
PS C:\Users\Karim\Desktop\Workspaces\Lean Workspace> leanproject -h
Usage: leanproject [OPTIONS] COMMAND [ARGS]...

  Command line client to manage Lean projects depending on mathlib. Use
  leanproject COMMAND --help to get more help on any specific command.

Options:
  -u, --from-url TEXT   Override base url for olean cache.
  -f, --force-download  Download olean cache without looking for a local
                        version.

  --no-lean-upgrade     Do not upgrade Lean version when upgrading mathlib.
  --debug               Display python tracebacks in case of error.
  --version             Show the version and exit.
  -h, --help            Show this message and exit.

Commands:
  add-mathlib        Add mathlib to the current project.
  build              Build the current project.
  check              Check mathlib oleans are more recent than their sources
  clean              Delete all olean files
  decls              List declarations seen from this project If no file...
  delete-zombies     Delete zombie oleans, .olean files with no matching...
  get                Clone a project from a GitHub name or git url.
  get-cache          Restore cached olean files.
  get-mathlib-cache  Get mathlib .lean and .olean files, without upgrading.
  global-install     Install mathlib user-wide.
  global-upgrade     Upgrade user-wide mathlib
  hooks              Setup git hooks for the current project.
  import-graph       Write an import graph for this project.
  mk-all             Creates all.lean importing everything from the project.
  mk-cache           Cache olean files.
  new                Create a new Lean project and prepare mathlib.
  pr                 Prepare to work on a mathlib pull-request on a new...
  rebase             On mathlib, update master, get oleans and rebase...
  set-url            Set the default url where oleans should be fetched.
  upgrade-mathlib    Upgrade mathlib (as a dependency or as the main...

**Comment by Kha** [comment]
You're still using leanproject, not elan.

**Comment by Kariiem** [comment]
The output
PS C:\Users\Karim\Desktop\Workspaces\Lean Workspace> elan init test
error: Found argument 'init' which wasn't expected, or isn't valid in this context

USAGE:
    elan.exe [FLAGS] <SUBCOMMAND>

For more information try --help

**Comment by Kariiem** [comment]
Please consider re-opening the issue.

**Comment by Kha** [comment]
Sorry, I meant to call `leanpkg init test` directly, which should use the `leanpkg` binary provided by elan. In any case, elan itself seems to be working for you now.

**Comment by Kariiem** [comment]
`leanpkg init test` was working before I updated elan (and before openning the isssue at lean4 repo) and it still works now. Actually, I had no problems with elan. My problem is with mathlibtools, the one called `leanproject`, as you have seen so far it doesn't work with lean4 (I switched to leanprover-community-lean-3.24.0 and it worked). 
However the project-file (source) tree seemed to be indifferent, only a src folder, with its path in the leanpkg.path, so I can do it myself. Also, there is something irritating in using `leanproject new project-name` (with leanprover-community-lean-3.24.0  ), it keeps downloading mathlibtools into the local directory of the project, this useless for a beginner like me.
Is there is anyway I can use the global mathlibtools installed at `C:\Users\Karim\.lean\_target\deps\mathlib`? Thanks in advance.
